joomla/joomla-cms
# [4.0] Add display field attibute

Pull Request for Issue  #32487 .

### Summary of Changes
Because the 'class'  XML attribute adds the class to the field itself rather than the outside container, it is not suitable for the classes added to  #32422. This PR adds a `parentclass` attribute which adds a class to the outside `control-group` div.


### Testing Instructions
Ensure https://github.com/joomla/joomla-cms/pull/32422 works as described (updated to match)



### Documentation Changes Required
